languages/javascript/concepts/equality/about.md

```diff
@@ -0,0 +1,13 @@
+Javascript uses `==`, `===` and `Object.is()` to compare the equality in different instances
```

Comment:
This should have a period at the end. I also have to wonder about the use of instances here. When I read this I am wondering what instances are being referred to. Is this instance for use in the sentence or are you talking about instances from the programming level perspective?